app/settings.py

```diff
@@ -28,7 +28,12 @@
 FTP_HEARTBEAT_FOLDER = os.getenv('FTP_HEARTBEAT_FOLDER', '/heartbeat')
 
 RABBIT_QUEUE = os.getenv('RABBITMQ_QUEUE', 'sdx-survey-notifications')
+RABBIT_DELAY_QUEUE = os.getenv('RABBIT_DELAY_QUEUE', 'sdx-delayed-survey-notifications')
```

Comment:
I think the delay queue should be named `sdx-survey-notifications_delay` to be consistent with the sdx-collect delay queue naming (putting delay at the end means the delay queue appears next to the main queue when listed alphabetically)